src/main/java/dumshenko/daniil/Square.java

```diff
@@ -0,0 +1,25 @@
+package dumshenko.daniil;
+
+public class Square extends Rectangle {
+
+    private double squareWidth;
+    private double squareHeight;
+
+    public Square(FigureType typeOfFigure, double squareWidth, double squareHeight) {
+        super(typeOfFigure, squareWidth, squareHeight);
+        this.squareWidth = squareWidth;
+        this.squareHeight = squareHeight;
+    }
```

Comment:
The square constructor must receive only one variable `size` as we know that `width = height = size` by geometrical definition of the Square shape.
```suggestion
    public Square(FigureType typeOfFigure, double size) {
        super(typeOfFigure, size, size);
    }
```